src/Kovan-DssSpell.sol

```diff
@@ -29,44 +27,45 @@ contract SpellAction {
     //  against the current release list at:
     //     https://changelog.makerdao.com/releases/kovan/active/contracts.json
 
-    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+    ChainlogAbstract constant CHANGELOG =
+        ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
+
+    address constant MCD_IAM_AUTO_LINE  = 0x0D0ccf65cED62D6CfC4DA7Ca85a0f833cB8889E4;
 
-    address constant MCD_ADM            = 0x27E0c9567729Ea6e3241DE74B3dE499b7ddd3fe6;
-    address constant VOTE_PROXY_FACTORY = 0x1400798AA746457E467A1eb9b3F3f72C25314429;
+    // decimals & precision
+    uint256 constant public WAD         = 10 ** 18;
+    uint256 constant public RAY         = 10 ** 27;
+    uint256 constant public RAD         = 10 ** 45;
 
     function execute() external {
-        address MCD_PAUSE   = CHANGELOG.getAddress("MCD_PAUSE");
-        address FLIPPER_MOM = CHANGELOG.getAddress("FLIPPER_MOM");
-        address OSM_MOM     = CHANGELOG.getAddress("OSM_MOM");
+        address MCD_VAT = CHANGELOG.getAddress("MCD_VAT");
 
-        // Change MCD_ADM address in the changelog (Chief)
-        CHANGELOG.setAddress("MCD_ADM", MCD_ADM);
+        // Give permissions to the MCD_IAM_AUTO_LINE to file() the vat
+        VatAbstract(MCD_VAT).rely(MCD_IAM_AUTO_LINE);
 
-        // Add VOTE_PROXY_FACTORY to the changelog (previous one was missing)
-        CHANGELOG.setAddress("VOTE_PROXY_FACTORY", VOTE_PROXY_FACTORY);
+        // Set the Line to equal the sum of current ilk lines
+        VatAbstract(MCD_VAT).file("Line", 1_227_000_000 * RAD);
```

Comment:
I forgot about this. I'm still not sure about the need to update the global Line on every exec. I think we might be better off if global line is set to the sum of the max of all of the maximum collateral line values, but it may be to late to make that change.

Reply:
Yeah, let's leave this for the future. For now the module already increases the general `Line` with each `line` increase that was so far the implemented logic.